### DISCUSSION TITLE
Calendar Channels

### DISCUSSION THREAD

**Discussion by RaeASunshine** [stoatchat/discussions#573] [Feature Suggestions]
I run a gaming guild that does regularly scheduled events, and allows users to create events as well.

One thing we would find _extremely_ useful is an integrated event calendar in a channel.  Some Google Calendar looking page where users with permissions can create new events on the calendar and anyone can see it.  Users could be able to select "I'm Going" for the event to sign up, and maybe you could even have a thread-like channel tied to the event so members joining the event can chat.  Perhaps even create a voice channel on demand for people to join.  

Such a feature would be extremely valuable and would allow me to encourage my guild to move over to Revolt, instead of fiddle with home grown discord bots that don't quite work the way we want them to.

**Comment by rogue-agent** [comment]
This calendar should also support CalDAV and WebDAV so I can synchronise it with my other devices and if I want it should also show in Stoat, during calendar scheduling, that I'm not available during my other calendar events.
